tigerpath/majors_and_certificates/majors/AAS.json

```diff
@@ -65,7 +52,7 @@
       "name": "Track",
       "max_counted": 1,
       "min_needed": 1,
-      "explanation": "<p>At the end of their fall semester, juniors declare a subfield to pursue, selecting from the following courses:</p><ul><li>African American Culture and Life (AACL)</li><li>Global Race and Ethnicity (GRE)</li><li>Race and Public Policy (RPP)</li></ul><p>Four (4) courses must be taken in the chosen subfield, with two (2) additional courses as follows:</p><ul><li>If the chosen subfield is AACL or RPP, then the two (2) additional courses must be selected from the GRE courses.</li><li>If the chosen subfield is GRE, then one (1) must be an AACL course, and one (1) must be an RPP course.</li></ul><p>Students are permitted to take <em>up to two (2)</em> approved cognate courses in other departments. See the <a href=\"https://aas.princeton.edu/courses\" target=\"_blank\" rel=\"noopener\">departmental website</a> for the lists of courses and approved cognates by subfield.</p>",
+      "explanation": "<p>If not selected at the time the concentration was declared, at the end of their fall semester, juniors declare a subfield to pursue, selecting from the following courses:</p><ul><li>African American Culture and Life (AACL)</li><li>Global Race and Ethnicity (GRE)</li><li>Race and Public Policy (RPP)</li></ul><p>Four (4) courses must be taken in the chosen subfield, with two (2) additional courses as follows:</p><ul><li>If the chosen subfield is AACL or RPP, then the two (2) additional courses must be selected from the GRE courses.</li><li>If the chosen subfield is GRE, then one (1) must be an AACL course, and one (1) must be an RPP course.</li></ul><p>Students are permitted to take <em>up to two (2)</em> approved cognate courses in other departments. See the <a href=\"https://aas.princeton.edu/courses\" target=\"_blank\" rel=\"noopener\">departmental website</a> for the lists of courses and approved cognates by subfield.</p>",

```

Comment:
I don't think we should change this. The explanations are meant to be directly quotes from the website, and it looks like they still keep the "At the end of their fall semester" wording (on https://aas.princeton.edu/academics/undergraduate-program/concentration-requirements).